src/Controls/src/Core/Handlers/Items/Android/SimpleViewHolder.cs

```diff
@@ -42,7 +42,7 @@ public static SimpleViewHolder FromText(string text, Context context, bool fill
 			return new SimpleViewHolder(textView, null);
 		}
 
-		public static SimpleViewHolder FromFormsView(View formsView, Context context, Func<int> width, Func<int> height, ItemsView container)
+		public static SimpleViewHolder FromFormsView(View formsView, Context context, Func<double> width, Func<double> height, ItemsView container)
```

Comment:
These Func calls are returning the android pixel sizes so it's easier to work in doubles for things like double.positiveinfinity